src/stores/fileStore.ts

```diff
@@ -0,0 +1,40 @@
+import { observable, action, runInAction } from 'mobx';
+import { telescoperApi } from '../services/api';
+import { HL7File } from '../types';
+
+export interface IFileStore {
+  files: HL7File[];
+  setFiles(files: HL7File[]): void;
+  getFiles(): Promise<void>;
+}
+
+export class FileStore implements IFileStore {
+  // TODO remove mock data
+  @observable files: HL7File[] = [
```

Comment:
Now that we are we are able to retrieve files / messages from the backend, will we keep this for testing?

Reply:
Testing is indeed something we should do. I remove this mock data in #3 but you're right that we should have some type of mock data for testing. I haven't written any tests for this so far since it's been in rapid change. We should start writing tests soon though, and when we do, we should do some experimentation and figure out a nice way of including mock data like you suggest.